docs/sitegen.yaml

```diff
@@ -172,6 +172,14 @@ backend:
               items:
                 - includes:
                     - "se/websocket/*.adoc"
+            - title: "Reactive Operators"
+              pathprefix: "/se/operators"
+              glyph:
+                type: "icon"
+                value: "fa-filter"
+              items:
+                - includes:
+                    - "mp/reactive/*.adoc"
```

Comment:
@m0mus Not sure about this, didn't want to duplicate same doc, operators are exactly the same for both flavors

Reply:
@danielkec It can be done using includes.

Reply:
See https://asciidoctor.org/docs/user-manual/#include-directive

Reply:
Aha took me long sorry :man_facepalming:

Reply:
It's more a question than suggestion. Would it be better to put shared pieces in `shared` folder and reference them from `se` and `mp`? In this case it's easy to find what is shared.,